kani-compiler/src/codegen_cprover_gotoc/codegen/function.rs

```diff
@@ -445,4 +449,68 @@ impl<'tcx> GotocCtx<'tcx> {
             }
         }
     }
+
+    /// Set the solver for this proof harness
+    fn handle_kanitool_solver(&mut self, attr: &Attribute, harness: &mut HarnessMetadata) {
+        // Make sure the solver is not already set
+        if harness.solver.is_some() {
+            self.tcx
+                .sess
+                .span_err(attr.span, "Only one '#[kani::solver]' attribute is allowed per harness");
```

Comment:
Can you change the error messages to start with a lowercase letter? I've been doing this following the [diagnostic output style guide](https://rustc-dev-guide.rust-lang.org/diagnostics.html#diagnostic-output-style-guide) from `rustc`, and I'd like us to start applying a similar style (will post a PR after this review).

Reply:
Done